datumaro/plugins/mots_format.py

```diff
@@ -77,9 +78,12 @@ def _parse_items(self):
 
         for p in sorted(iglob(self._anno_dir + '/**/*.png', recursive=True)):
             item_id = osp.splitext(osp.relpath(p, self._anno_dir))[0]
+            image = None
+            image_path = images.get(item_id)
+            if image_path:
+                image = Image(path=image_path)
```

Comment:
```suggestion
            image = images.get(item_id)
            if image:
                image = Image(path=image)
```
Seems like a pretty common pattern

Reply:
Fixed it in all formats